apps/avifenc.c

```diff
@@ -18,15 +18,13 @@
 // for setmode()
 #include <fcntl.h>
 #include <io.h>
+#include <locale.h>
+#define WIN32_LEAN_AND_MEAN
+// Avoid the DEFAULT_QUALITY macro redefinition warning caused by including wingdi.h.
+#define NOGDI
+#include <windows.h>
 #endif
 
-#define NEXTARG()                                                     \
-    if (((argIndex + 1) == argc) || (argv[argIndex + 1][0] == '-')) { \
-        fprintf(stderr, "%s requires an argument.", arg);             \
-        goto cleanup;                                                 \
```

Comment:
Hmm... I think we had a bug here. We should set `returnCode` to 1 before `goto cleanup`.

Reply:
Indeed, fixed in https://github.com/AOMediaCodec/libavif/pull/1699. Other paths were wrong too.

Reply:
Thanks a lot!